autoload/neomake.vim

```diff
@@ -248,10 +248,12 @@ function! s:Make(options) abort
         endif
     endif
 
+    let title = join(map(copy(enabled_makers), 'v:val.name'), ', ')
```

Comment:
Looks pretty reasonable to me. However, no need to copy `enabled_makers` here since we aren't mutating it.